src/components/Committee/ProfileCard.jsx

```diff
@@ -0,0 +1,91 @@
+import {React,useState} from "react"
+import {Icon} from "@iconify/react"
+import img from "../Committee/images/psood.jpg"
+export default function ProfileCard() {
+     const people = [
+            {
+              id: 1,
+              name: "Devanshi Patel",
+              position: "EM Head",
+            },
+            {
+              id: 2,
+              name: "Riya Sharma",
+              position: "President",
+            },
+            {
+              id: 3,
+              name: "Jeet Davda",
+              position: "VP",
+            },
+          ]
+        
+          let [current, setCurrent] = useState(0)
+        
+          const studList = people.map((people) => {
+            return (
+              <div
+                key={people.id}
+                className={
+                  "xs:w-4/5 relative w-full rounded-2xl  py-8   drop-shadow-md sm:w-[85%] xl:px-11"
+                }>
+       
+      <section class=" w-full lg:w-1/2 px-5 py-5 text-center bg-gray-600 hover:bg-primary hover:text-black rounded-[2rem] duration-200 shadow-lg p-8">  
+     <div class="w-7/12 text-center max-w-md ml-auto mr-auto mt-8">   
```

Comment:
for fixing that `contact` try removing `w-7/12` and `max-w-md` class. I think it will work